kbfsmd/root_metadata_v3.go

```diff
@@ -645,11 +645,35 @@ func (md *RootMetadataV3) CheckValidSuccessorForServer(
 	return nil
 }
 
+// IsBackedByTeam returns true if md is for a TLF backed by a team. It could be
+// either a SingleTeam TLF or a private/public TLF backed by an implicit team.
+func (md *RootMetadataV3) IsBackedByTeam(extra ExtraMetadata) bool {
```

Comment:
Please place this above `MakeInitialRootMetadataV3` where it's used.

Reply:
Hmm that one uses the one defined on `tlf.Handle`. In the new version it's gonna be a different method though.